src/OrleansServiceBus/Providers/Streams/EventHub/EventHubConstants.cs

```diff
@@ -0,0 +1,12 @@
+namespace Orleans.ServiceBus.Providers
+{
+    internal class EventHubConstants
+    {
+        public readonly static string StartOfStream =
+#if NETSTANDARD
+            Microsoft.Azure.EventHubs.PartitionReceiver.StartOfStream;
+#else
+            Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.StartOfStream;
+#endif
+    }
```

Comment:
Duh, it is already internal, so unless there are other objections, I'll leave it as is.